les/peer.go

```diff
@@ -89,6 +91,7 @@ type peer struct {
 	// RequestProcessed is called
 	responseLock  sync.Mutex
 	responseCount uint64
+	invalidReq    uint32
```

Comment:
Just oooone tiny nitpick, can we name this `invalidCount`? That way it's consistent with `responseCount`

Reply:
Sure, done